### PR TITLE
First version of Write VIP

### PR DESCRIPTION
**Pull request by tpazderka**

### CODE REVIEW

File: cz_nia/functions.py [hunk 1]
```diff
@@ -106,3 +106,15 @@ def get_pseudonym(settings: CzNiaAppSettings, user_data: Dict[str, Any]) -> str:
     message = IdentificationMessage(user_data)
     body = _call_submission(settings, transport, sub_assertion, message)
     return message.unpack(body)
+
+
+def write_vip(settings: CzNiaAppSettings, data):
```
Comment (ziima) [inline cz_nia/functions.py]: `register_authenticator`?

Comment (tpazderka) [reply cz_nia/functions.py]: It allows change as well (deactivation) so probably `write_authenticator`?

Comment (ziima) [reply cz_nia/functions.py]: Looks good.

Comment (tpazderka) [reply cz_nia/functions.py]: Renamed

**Pull request merged**